test/integration/targets/docker_swarm_service/tasks/tests/misc.yml

```diff
@@ -0,0 +1,125 @@
+---
+- block:
+    - name: Create a Swarm cluster
```

Comment:
I think creating the swarm cluster should be moved to line 18 of test/integration/targets/docker_swarm_service/tasks/main.yml, so that it is done inside the `block:` but before the tests are executed. Otherwise if someone adds a test with comes alphabetically before `misc`, that one won't have a swarm cluster to work with.

Reply:
I was thinking that each test would be responsible of creating their own swarm but it feels kind of redundant. I'll remove the creating / destruction of swarms from both `options.yml` and `misc.yml`.